weis/dlc_driver/dlc_generator.py

```diff
@@ -51,19 +52,28 @@
 
     'wave_model': ("SeaState","WaveMod"),
     
+    'startup_mode': ("DISCON_in","SU_Mode"),
```

Comment:
Someday, I'm not sure if it's today, I'd like to standardize the key/input names here to follow a more pythonic naming convention.